package.json

```diff
@@ -23,6 +23,7 @@
     "nightwatch": "nightwatch -c config/nightwatch.js",
     "nightwatch-sauce": "nightwatch -c config/nightwatch-sauce.js",
     "nightwatch-visual": "node src/platform/testing/visual-regression/index.js",
+    "install": "./script/install-repos.sh",
```

Comment:
One thing to note: developers run `yarn install` regularly, because it brings your dependencies up to date if there have been changes. Looks like this script fails if the repos are already cloned.

Reply:
Ok, no problem... I'll change the script to not fail if dirs are already present

Reply:
I think we should rename this and document this behavior in the readme as a separate command. Cloning repos outside of package directory is not generally expected behavior for an install script. I think the install script should _only_ install  _this_ package's dependencies into `node_modules`.

We should also make sure that any changes to the install include changes to the documentation: https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/

Reply:
I'm with @rianfowler. If I run `yarn install`, this not only updates the dependencies, but runs this scrip to install the repos. Cloning repos outside of the working directory as a side-effect of updating my dependencies isn't great.

What do you think about changing this to `install-repos`? Can you update your `composer` build script to run both `install` and `install-repos`? (I remember basically nothing about `composer` from my PHP days. :sweat_smile:)

Reply:
> Can you update your composer build script to run both install and install-repos?

Absolutely, composer scripts are arrays, I can add extra commands.

I just thought that, since WEB does depend on this repo to work, it would be better to include it in the WEB repo. But if you think differently, I can work around it, no problem.

Reply:
Is this still needed?

Reply:
Only one way to find out :)  ...

Does `yarn install` and `npm build` work if the content repo is not there?

Reply:
Ok, after chatting with @cvalarida the content repo is still required, but they still don't want to use the `yarn install` command to do it.

So, we will: 

- [ ] Create a new `yarn install-content` command.
- [ ] Remove the Shell script.
- [ ] Add a call to `yarn install-content` before `yarn install` in our build process in `composer.json`

Reply:
@jonpugh Sounds about right! Thanks for this! :raised_hands:

Reply:
Actually, @cvalarida would you be able to handle that todo list for us?

I'm not personally versed in Yarn yet, I feel like this would take you about 30 seconds...  :)

LMK!